TODO/the-ultimate-guide-to-creating-a-mobile-application.md

```diff
@@ -8,191 +9,191 @@
+用户调研意味着对核心目标人群心理特性的更深入研究，不同因素的影响如颜色、样式选择、表情的交互逻辑和分组体验、信息源和创造性的表现都可以让用户参与进来并提高其活跃度。市场调研意味着浏览市场部分，主要是对竞品的深入研究。大胆视觉设计和原创性以及独到的风格都可以使产品鹤立鸡群并牢牢吸引目标人群的注意力。
 
-Famous guru of advertising David Ogilvy emphasized the great importance of research for creating effective result: «Advertising people who ignore research are as dangerous as generals who ignore decodes of enemy signals.» Time has changed the means, goals and technologies, still vital role of research established even stronger. Neglecting the research stage and relying only on their creative intuition, experience and talent, designers risk failing this task as they will not know the conditions of the app functioning and will not be able to make it efficient, user-friendly and original.
+广告业知名专家 David Ogilvy 强调了调研对有效成果的重要性：忽略调研的广告人就像忘记解密敌方通信一样愚蠢。方法、目标和技术随时间而改进，但调研扮演的角色缺越来越重要。忽视调研阶段而只依赖于他们的创造性直觉、经验和天赋的话，设计师有失败的危险了，因为他们不会知道应用工作状况，也无法使其变得高效、用户友好而且独特。
 
 ![](https://cdn-images-1.medium.com/max/800/0*_zW-wAx5G6vZLeTe.jpg)
 
-### UX Wireframing
+### UX 线框图
 
-It is the process of creating general structure of the designed application or website. It’s usually accomplished via the set of schematic screens or pages of low or middle level of fidelity. The aim of this stage is setting clear and orderly structure of all the layout, transitions and interactions on the basis of user’s problems and pains which the product is going to solve.
+这个阶段是建立设计应用或网站通用结构的。通常情况下会完成一套示意性的低或中清晰度的草稿（数字或纸张形式）。目标是使所有要解决的用户基础布局、过渡和交互问题变得清晰而有层次性。
 
-In one of [our previous articles](http://tubikstudio.com/design-faq-platform-customers-set-two/) we provided a bit of metaphor on that. When we think about building the house, for example, we usually mean the process of physical appearance of the construction rather than tons of projects, drawings and calculations made on paper. And yes, physically it’s possible to build the house without any project as well as it’s possible to create the interface out of thin air. However, in this case you shouldn’t be surprised if one day the house will crack and collapse without any visible reasons as well as the app looking amazing and stylish won’t bring you any loyal users. If you want to have a reliable house, a durable mechanism, a powerful application or a highly-functional website, the recipe is the same — take your time for thorough planning and projecting. This is not going to waste your time, vise versa, it will save your time you would otherwise have to spend on redesign and attempts to find out why your product doesn’t work properly.
+在 [我们先前的一篇文章中](http://tubikstudio.com/design-faq-platform-customers-set-two/) 提供了一些这方面的示例。举个例子，当我们开始盖房子时，我们首要考虑的是房子的物理结构问题而不是一堆计划表、草图和各种计算。当然也可以什么都不考虑直接盖个空中楼阁。那当它有一天毫无征兆地塌了的时候麻烦你合上那脱臼的下巴。华而不实的应用并不会吸引到任何死忠用户。如果你想有一幢可靠的房子，一套可持续运转的机器，一个强大的应用又或者是个高度集成的网站，原理都一样——多花点时间来设计和计划。这并不会浪费时间，恰恰相反，这将节省你在未来可能花在重新设计和找问题上的时间。 
```

Comment:
When we think about building the house, for example, we usually mean the process of physical appearance of the construction rather than tons of projects, drawings and calculations made on paper.

我们首要考虑的是房子的物理结构问题而不是一堆计划表、草图和各种计算。

这里的意思应该不是 否定 先想到 一堆计划表、草图和各种计算，而说的是我们容易忽略“一堆计划表、草图和各种计算”。

所以我觉得应该翻成：
When we think about building the house, for example, we usually mean the process of physical appearance of the construction rather than tons of projects, drawings and calculations made on paper. And yes, physically it’s possible to build the house without any project as well as it’s possible to create the interface out of thin air.

当我们想到要盖一栋房子的时候，我们往往想到的是其物理上从无到有的过程，而不是一大堆的计划表、草图和各种计算
。诚然，理论上说我们可以不做任何计划就搭好一栋房子，凭空想象一个用户界面也是一个道理（也是可能的）。